src/pages/SearchPage.js

```diff
@@ -116,88 +115,71 @@ class SearchPage extends Component {
         return sections;
     }
 
-    searchRendered() {
+    const searchRendered = () => {
         Timing.end(CONST.TIMING.SEARCH_RENDER);
         Performance.markEnd(CONST.TIMING.SEARCH_RENDER);
     }
 
-    updateOptions() {
-        const {recentReports, personalDetails, userToInvite} = OptionsListUtils.getSearchOptions(
-            this.props.reports,
-            this.props.personalDetails,
-            this.state.searchValue.trim(),
-            this.props.betas,
-        );
-        this.setState({
-            userToInvite,
-            recentReports,
-            personalDetails,
-        });
+    useEffect(() => {
```

Comment:
Could you please group all `useEffects` in one place instead of having them in random places across the component.